### ISSUE TITLE
Cannot resolve @typeparam

### ISSUE THREAD

**Issue by kakins** [aspnet/Blazor#1552]
I have Blazor 0.6.0 installed, using the VS template.  .NET Core SDK 2.1.402.  I see this, both with ReSharper enabled and disabled.

![image](https://user-images.githubusercontent.com/14915247/46832614-1fe40780-cd6c-11e8-9cc5-c7ee4a70d848.png)

**Comment by danroth27** [comment]
Hmm, strange. I'm not able to reproduce this. @rynowak Any ideas?

**Comment by rynowak** [comment]
I don't have any off the cuff ideas. Can you confirm that other blazor features work in the editor? `@layout` for instance. Can you confirm the version of the blazor extension in the `About...` dialog

![image](https://user-images.githubusercontent.com/1430011/46841909-6d1da480-cd7f-11e8-901d-de40e6d081ad.png)

**Comment by rynowak** [comment]
One other thing to check. Did package restore complete successfully for the project?

**Comment by rynowak** [comment]
Can you please report back about `@layout`? `@inject` works the same way in MVC as it does in Blazor. I'm curious if the editor thinks this file is in an MVC project.

**Comment by kakins** [comment]
I created the project from the VS template, and opened `_ViewImports.cshtml`.  I see no syntax errors.  I also retyped the `@layout` line and it still appears to be working fine.

![image](https://user-images.githubusercontent.com/14915247/46961844-4dc89500-d067-11e8-96f7-6332b5e35caa.png)

**Comment by kakins** [comment]
By the way, just in case, here is my .csproj
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
    <RunCommand>dotnet</RunCommand>
    <RunArguments>blazor serve</RunArguments>
    <LangVersion>7.3</LangVersion>
  </PropertyGroup>

  <ItemGroup>
    <BlazorLinkerDescriptor Include="linker.xml" />
  </ItemGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Blazor.Browser" Version="0.6.0" />
    <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="0.6.0" PrivateAssets="all" />

    <DotNetCliToolReference Include="Microsoft.AspNetCore.Blazor.Cli" Version="0.6.0" />
  </ItemGroup>

</Project>
```

Also, here is the linker.xml I'm using

```
<linker>
  <assembly fullname="mscorlib">
    <!-- Preserve all methods on WasmRuntime, because these are called by JS-side code
    to implement timers. Fixes https://github.com/aspnet/Blazor/issues/239 -->
    <type fullname="System.Threading.WasmRuntime" />
  </assembly>
  <assembly fullname="System.Core">
    <!-- This is required by JSon.NET and any expression.Compile caller -->
    <type fullname="System.Linq.Expressions*" />
  </assembly>
  <!-- Name of the entry point assembly -->
  <assembly fullname="BlazorTest" />
</linker>
```

**Comment by danroth27** [comment]
@kakins Can you share the project with us?

**Comment by nolisj** [comment]
I'm having the same problem. All other directives are working (@page, @inherits, etc). I've tried creating from various templates just to isolate:  a Razor View, Razor Page, Razor Layout. My VS2017 is version 15.9.0 Preview 4. 

In my case though, it turns out this is an issue only when I have ReSharper (v2018.2.3) active. If I disable ReSharper, it works properly.

**Comment by alexDevBR** [comment]
I had the same issue and was able to resolve it by uninstalling Blazor language services through the VS installer and reinstalling it in Extensions and Updates.

**Comment by nolisj** [comment]
@alexDevBR, I'm interested to know if your working setup included a ReSharper. Does it?

**Comment by alexDevBR** [comment]
@nolisj no, I don't have ReSharper. I also upgraded to 15.9.1, maybe that also helped, but what solved it was reinstalling Blazor language services. I reinstalled it after the upgrade, but maybe reinstalling it before would have solved the problem too.
